scripts/zkai_d128_residual_add_proof_input.py

```
@@ -0,0 +1,679 @@
+        "range_policy": RANGE_POLICY,
+        "input_activation_commitment": INPUT_ACTIVATION_COMMITMENT,
+        "residual_delta_commitment": RESIDUAL_DELTA_COMMITMENT,
+        "residual_delta_scale_divisor": 512,
+        "non_claims": NON_CLAIMS,
+        "proof_verifier_hardening": PROOF_VERIFIER_HARDENING,
+        "next_backend_step": NEXT_BACKEND_STEP,
+        "validation_commands": VALIDATION_COMMANDS,
+    }
+    for field, expected in constants.items():
+        if payload.get(field) != expected:
+            raise D128ResidualAddInputError(f"payload field mismatch: {field}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)